core/src/main/res/layout/main_activity_content.xml

```diff
@@ -20,14 +20,15 @@
 
       <include layout="@layout/layout_toolbar" />
 
-      <org.kiwix.kiwixmobile.core.main.AnimatedProgressBar
+      <ProgressBar
         android:id="@+id/activity_main_progress_view"
         android:layout_width="match_parent"
         android:layout_height="@dimen/progress_view_height"
+        android:max="100"
+        android:progress="1"
+        android:indeterminate="false"
         app:backgroundColor="?colorOnSurface"
-        app:bidirectionalAnimate="false"
-        app:layout_constraintBottom_toBottomOf="@+id/toolbar"
-        app:progressColor="@color/blue400" />
```

Comment:
if memory serves this is important, we very much wanted the progressbar to be this color

Reply:
Yes..😅

Reply:
You may want to consider a ThemeOverlay here, progressbars are a bit dickish about setting their color <21